marlowe-playground-client/src/MainFrame/State.purs

```diff
@@ -532,13 +553,19 @@ handleGistAction ::
 handleGistAction settings PublishGist = do
   description <- use _projectName
   let
+    toNothing :: forall a. Eq a => Monoid a => Maybe a -> Maybe a
```

Comment:
The name's odd here. I would expect `toNothing = const Nothing`.

Erm...`pruneEmpty`?

Reply:
`removeEmpty`?

Reply:
yeah, I was looking for a better name, thanks.